### DISCUSSION TITLE
Support for fixed migration dates

### DISCUSSION THREAD

**Discussion by jkeiski** [spatie/laravel-package-tools#63] [Ideas]
Currently when publishing migrations, package uses current timestamp when copying migration files from package migrations to application migrations. Usually this is a good behaviour, but I have a use case where application migrations should build upon package migrations so using fixed date could help to make package migrations run first. 

I solved this for my immediate need by forking it and making a minor change to check if migration file name starts with numeric value and if so then don't append extra date when copying the file. I had no time to find out why console output still shows double date file names or write tests so I didn't make PR for it.

**Comment by freekmurze** [comment]
I feel like what we have now covers the 90% use case, so we'll leave it at this.
